bigbang/w3crawl.py

```diff
@@ -26,7 +26,7 @@ class W3cMailingListArchivesParser(email.parser.Parser):
     archive pages generated by W3C's mail archives system, based on Hypermail.
     """
 
-    parse = None
+    # parse = None <- is this supposed to redefine parse from line 20?
```

Comment:
`parse = None` should remain, I believe it's the recommended way to indicate that this subclass of `email.parser.Parser` doesn't implement the parse method (only `parsestr`). Line 20 is actually unnecessary, but a separate issue that doesn't need to be resolved in this PR.

Reply:
Ok, have uncommented it again :+1: